frontend/pages/club/[club]/index.tsx

```diff
@@ -247,6 +247,14 @@ const ClubPage = ({
             </StyledCard>
           )}
           <Events data={events} />
+          {isClubFieldShown('signature_events') && club.signature_events && (
```

Comment:
This check will actually pass if `club.signature_events` is an empty string (`""`). You need to check `club.signature_events.length` in order to properly catch this edge case.